src/System.Private.ServiceModel/tools/scripts/ConfigHttpsPort.ps1

```diff
@@ -0,0 +1,15 @@
+﻿$FriendlyName = "WCF Bridge - Machine certificate generated by the CertificateManager"
+
```

Comment:
Not blocking -- for future consideration, we may need to parameterize these port #'s.  If they sometimes come from environment variables and sometimes are hard-coded, it will be hard to track down problems.